pkg/bloomcompactor/bloomcompactor_test.go

```diff
@@ -0,0 +1,134 @@
+package bloomcompactor
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/grafana/dskit/flagext"
+	"github.com/grafana/dskit/kv/consul"
+	"github.com/grafana/dskit/ring"
+	ww "github.com/grafana/dskit/server"
```

Comment:
There is no other imported module named `server`

```suggestion
	"github.com/grafana/dskit/server"
```

Reply:
Thank you!